tests/qos/test_buffer.py

```diff
@@ -2038,7 +2038,7 @@ def test_exceeding_headroom(duthosts, rand_one_dut_hostname, conn_graph_facts, p
                                                  'Failed to remove buffer profile .* with type BUFFER_PROFILE_TABLE',
                                                  'doTask: Failed to process buffer task, drop it'])
         logging.info('[Find out the longest cable length the port can support]')
-        cable_length = 300
+        cable_length = int(original_cable_len[:-1])
```

Comment:
Why is the indexing needed for cable len?

Reply:
@neethajohn to remove the suffix ’m’: ‘5m’ => ‘5’